modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/LagomScalaApiCodegen.java

```diff
@@ -0,0 +1,268 @@
+package io.swagger.codegen.languages;
+
+import io.swagger.codegen.*;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.*;
+
+public class LagomScalaApiCodegen extends AbstractScalaLagomCodegen implements CodegenConfig {
```

Comment:
I would suggest naming it as "ScalaLagomServerCodegen" following the naming convention of other server generator.

Reply:
Done